badges/badgesAvtaar.css

```diff
@@ -0,0 +1,90 @@
+.avtaar-badges{
+    display:flex;
+    gap:1rem;
+    flex-direction: row;
```

Comment:
You don't have to explicitly mention `flex-direction: row` as by default `display: flex` takes row. Just to make code more concise.